### PR TITLE
MADE GPA WORK

### PR DESCRIPTION
**Pull request by LadyAuroralite**

### CODE REVIEW

**Review by HeyBanditoz — CHANGES_REQUESTED** [review]
we are not doing this. https://github.com/herrimanpandaproject/pandaproject-mobile1/pull/10#discussion_r287582029

File: app/src/main/java/com/instructure/template/projectCodeHere/layouts/FragmentProfilePage.java [hunk 1]
```diff
@@ -109,11 +109,35 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
                 int count=0;
                 double total=0;
                 for(GetEnrollments.EnrollmentResponse i : g) {
-                    total+=i.getGrades().getCurrent_score();
+                    if(i.getGrades().getCurrent_score()>=93) {
```
Comment (LeaHiHi) [inline app/src/main/java/com/instructure/template/projectCodeHere/layouts/FragmentProfilePage.java]: Your making O(12N) api calls when you could make 1

**Review by LeaHiHi — CHANGES_REQUESTED** [review]

**Review by HeyBanditoz — APPROVED** [review]
It could be better, somehow.

**Comment by LadyAuroralite** [comment]
made it only call the api once per class aswell as no waterfall if

**Review by LeaHiHi — APPROVED** [review]

**Pull request merged**